src/App.js

```diff
@@ -54,7 +54,6 @@ export default function App() {
           <CssBaseline />
           <ResponsiveAppBar />
           <Routes>
-            {/* TODO: put a home page for root url */}
```

Comment:
if not done in here, it would be nice to have an issue, and associate the TODO comment with the issue number, this way we don't forget about it :)